src/api/api.cc

```diff
@@ -11376,6 +11392,30 @@ extern "C" DLLEXPORT bool V8RecordReplayAreAssertsDisabled() {
   return recordreplay::AreAssertsDisabled();
 }
 
+bool recordreplay::ShouldReportPerformanceEvent(uint32_t kind) {
```

Comment:
I was trying to make a push for this in the past, but... can we start moving concrete subset of features into their own files? I really don't think, keeping on piling things into `api.cc` is a good move moving forward 🤷‍♀️

Reply:
I agree another file would be nice but this isn't really the time or place to address that.